packages/middleware/__tests__/integration/createServer.spec.ts

```diff
@@ -141,4 +153,35 @@ describe("[Integration] Create server", () => {
       },
     });
   });
+
+  it("should make a call to a namespaced method", async () => {
+    expect.assertions(2);
+
```

Comment:
```suggestion
```
It's for assertions in `catch` block, to fail tests if `try` block didn't throw as expected.